src/common/utils/getDropDownItems.js

```diff
@@ -0,0 +1,17 @@
+import { get } from 'lodash';
+
+const EMPTY_DROPDOWN_ITEM = { label: '', value: '' };
+
+export const getDropDownItems = (resource, propName, isRequired) => {
```

Comment:
in suggestion it was `dropdown.js`, because you might export not only `getDropDownItems` from there.